cove_project/settings.py

```diff
@@ -149,8 +149,8 @@
     "schema_host": None,
     "schema_version_choices": OrderedDict(
         (  # {version: (display, url)}
-            ("1.0", ("1.0", "https://standard.open-contracting.org/schema/1__0__3/")),
-            ("1.1", ("1.1", "https://standard.open-contracting.org/schema/1__1__5/")),
+            ("1.0", ("1.0", "https://standard.open-contracting.org/1.0/{lang}/")),
+            ("1.1", ("1.1", "https://standard.open-contracting.org/1.1/{lang}/")),
```

Comment:
@jpmckinney This change means that cove-ocds will be updated as soon as the 1.1 site is updated, rather than waiting to be pointed at a new patch directory. Is that okay?

Reply:
I think so. New patch releases should not break CoVE. For 1.1.5, I had to update tests, and I had to change the URL for `schema_codelists`, but this URL should not change again.

Reply:
You can see the 1.1.5 changes in this range. Similar changes were made to lib-cove-ocds. 428bacbb79834329ed62dd4a15186f439ea66fce..d77a3a15916c159c319d5bbfe84d515511cac661